tests/test_user_scoped_client.py

```
@@ -0,0 +1,119 @@
+            resp = tc.get("/outfits", headers={"Authorization": "Bearer any.jwt"})
+
+    assert resp.status_code == 200
+    admin_factory.assert_not_called()
+
+
+class _FakeRLSQuery:
+    """Models PostgREST+RLS: rows are pre-filtered to the bound user on the
+    server, so client-side `.eq("user_id", ...)` is irrelevant to isolation.
+    """
+
+    def __init__(self, visible_rows):
```

Ruff (0.15.12)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)